client/src/components/App.jsx

```diff
@@ -1,15 +1,27 @@
 import React from 'react';
+import styled from 'styled-components';
+import Calendar from './Calendar.jsx';
+import axios from 'axios';
+
+const Title = styled.header`
+  font-size: 30px;
+  font-family: arial;
+  margin-bottom: 10px;
+`;
 
 class App extends React.Component {
-  constructor() {
-    super();
-  }
+  constructor(props) {
+    super(props);
+  };
```

Comment:
nit: Remove semicolon from here to be consistent with the rest of your code.

Reply:
Is that semicolon inconsistent? I thought I had it everywhere else in my code?